README.md

```diff
@@ -0,0 +1,24 @@
+# Median housing value prediction
+
+The housing data can be downloaded from https://raw.githubusercontent.com/ageron/handson-ml/master/datasets/housing/housing.csv. The script has codes to download the data. We have modelled the median house value on given housing data. 
+
+The following techniques have been used: 
+
+ - Linear regression
+ - Decision Tree
+ - Random Forest
+
+## Steps performed
+ - We prepare and clean the data. We check and impute for missing values.
+ - Features are generated and the variables are checked for correlation.
+ - Multiple sampling techinuqies are evaluated. The data set is split into train and test.
+ - All the above said modelling techniques are tried and evaluated. The final metric used to evaluate is mean squared error.
+
+## To excute the script
+```conda env create -f env.yml```
+```conda activate mle-dev```
+```python nonstandardcode.py```
+```conda install pandas```
+```conda install matplotlib```
+```python nonstandardcode.py```
+```conda env export> env.yml```
```

Comment:
1. Since you are already creating the environment from a YAML file which contains pandas, matplotlib and other libraries, there is no need to install these libraries again.
2. No need to re-run nonstandardcode.py script
3. You don't need to export the environment since we already have it available.

You should remove these lines, as it might create confusion for the receiver of this document.

Since these lines don't affect the final intention of the assignment, I am approving the PR. 

But please, do such kind of quality checks before submitting your final deliverables (this will help the team members in your project).

Reply:
@ankushbhardwaj-tiger